lib/tournament.ex

```diff
@@ -1,4 +1,21 @@
 defmodule Tournament do
+  @separator_input ";"
+  @separator_output " | "
+  @team "Team"
+  @matches "matches played"
+  @win "win"
+  @loss "loss"
+  @draw "draw"
+  @points "points"
+
+  @value_default 0
+  @points_win 3
+  @points_draw 1
+  @points_loss 0
+
+  @pad_team 30
+  @pad_value 2
```

Comment:
All "magic" strings and numbers moved to a central configurable location. A Refactoring by Robert C Martin thing I believe?

Anyways, a floating string in a function saying "win" is a red flag to me and therefore I tend to extract them. Maybe even better to extract some to their own globals file, depending on where the data incoming is. **For example, _should_ this file be worried about semicolon separators? Possibly not.**

---

Side benefit: It's easy to see what the points are rewarded for wins/draws/losses at the top of the file now.